src/main/java/de/castcrafter/travel_anchors/block/RenderTravelAnchor.java

```diff
@@ -31,64 +35,83 @@
 
     public static final ResourceLocation ANCHOR_MODEL = new ResourceLocation(TravelAnchors.getInstance().modid, "block/travel_anchor");
     private static IBakedModel ANCHOR_MODEL_BAKED = null;
+    private static IRenderTypeBuffer buffer;
 
     public RenderTravelAnchor(TileEntityRendererDispatcher rendererDispatcherIn) {
         super(rendererDispatcherIn);
     }
 
     @Override
-    public void render(TileTravelAnchor tile, float partialTicks, @Nonnull MatrixStack matrixStack, @Nonnull IRenderTypeBuffer buffer, int combinedLight, int combinedOverlay) {
-        //noinspection deprecation
-        if (tile.getMimic() == null || tile.getMimic().getBlock() == tile.getBlockState().getBlock() || tile.getMimic().isAir()) {
-            IVertexBuilder vertexBuffer = buffer.getBuffer(RenderTypeLookup.func_239220_a_(tile.getBlockState(), false));
-            //noinspection deprecation
-            Minecraft.getInstance().getBlockRendererDispatcher().getBlockModelRenderer()
-                    .renderModelBrightnessColor(matrixStack.getLast(), vertexBuffer, tile.getBlockState(),
-                            ANCHOR_MODEL_BAKED, 1, 1, 1, combinedLight, OverlayTexture.NO_OVERLAY);
-        } else {
-            Minecraft.getInstance().getBlockRendererDispatcher().renderBlock(tile.getMimic(), matrixStack, buffer, combinedLight, OverlayTexture.NO_OVERLAY, tile.getModelData());
-        }
```

Comment:
das muss bleiben, sonst wird der Block nicht gerendert, wenn der RenderType = EntityBlock_animated
nicht vergessen this.buffer = RenderBuffer davor zu schreiben sonst chrashed es